bundles/org.openhab.core.automation/src/main/java/org/openhab/core/automation/handler/TimeBasedConditionHandler.java

```diff
@@ -30,5 +30,5 @@ public interface TimeBasedConditionHandler extends ConditionHandler {
      * @param time The time to check.
      * @return <code>true</code> if and only if the given time satisfies this condition.
      */
-    public abstract boolean isSatisfiedAt(ZonedDateTime time);
+    abstract boolean isSatisfiedAt(ZonedDateTime time);
```

Comment:
Could the `abstract` keyword be removed as well by the script?

Reply:
Yes it worked, though I had to make one manual correction. See the new commit.